applications/FemToDemApplication/python_scripts/MainCoupling2WayFemDem.py

```diff
@@ -0,0 +1,265 @@
+
+
+import KratosMultiphysics
+import KratosMultiphysics.FemToDemApplication as KratosFemDem
+import KratosMultiphysics.StructuralMechanicsApplication as KratosSMA
+
+import KratosMultiphysics.FemToDemApplication.MainDEM_for_coupling as DEM
+import KratosMultiphysics.FemToDemApplication.MainFEM_for_coupling as FEM
+import KratosMultiphysics.FemToDemApplication.FEMDEMParticleCreatorDestructor as PCD
+import math
+import os
+import KratosMultiphysics.MeshingApplication as MeshingApplication
+import KratosMultiphysics.MeshingApplication.mmg_process as MMG
+import KratosMultiphysics.DEMApplication as KratosDEM
+import KratosMultiphysics.DemStructuresCouplingApplication as DemFem
+import KratosMultiphysics.FemToDemApplication.MainCouplingFemDem as MainCouplingFemDem
+
+def Wait():
+    input("Press Something")
+
+#============================================================================================================================
+class MainCoupled2WayFemDem_Solution(MainCouplingFemDem.MainCoupledFemDem_Solution):
```

Comment:
Hi @AlejandroCornejo . I strongly recommend to atomize the methods in the class `MainCouplingFemDem.MainCoupledFemDem_Solution`
I would take pieces of code which can be encapsulated in one function and move them to a method. Then, when you derive, like you are doing here, you will duplicate a lot less lines.

Reply:
You're totally right. I'll do another PR cleaning everything